.gitignore

```diff
@@ -0,0 +1,4 @@
+*.tfstate
+
+.idea
```

Comment:
I would add this to your local ignore in your machine, not for all the team.

Take a look at this: https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
Makes sense. Thx